.sequelizerc

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
Why are you still using ES5 in your codes?

Reply:
You need to be professional in your commenting. That statement should have been 'Use ES6'

Reply:
Ok, maybe that. have you corrected it?